packages/e2e-tests/specs/performance/post-editor.test.js

```diff
@@ -105,28 +105,43 @@ describe( 'Post Editor Performance', () => {
 			readFile( join( __dirname, '../../assets/large-post.html' ) )
 		);
 		await saveDraft();
-		let i = 5;
+		const draftURL = await page.url();
+
+		// Number of sample measurements to take.
+		const samples = 5;
+		// Number of throwaway measurements to perform before recording samples.
+		// Having at least one helps ensure that caching quirks don't manifest in
+		// the results.
+		const throwaway = 1;
+
+		let i = throwaway + samples;
 		while ( i-- ) {
```

Comment:
if only we had a construct that let us assign an initial counter value and specify how to bump it on each iteration 😆

Reply:
I suppose I could have refactored the code further as I was changing it 🙂